docroot/themes/humsci/humsci_basic/templates/menus/macros/secondary-nav-menu.twig

```diff
@@ -36,10 +36,10 @@
         #}
         {% set link_attributes = link_attributes.addClass('hb-secondary-toggler') %}
         {% set randomInt = random(0, 100) %}
-        <a{{ link_attributes }} id="{{ item.title|replace({' ': ''}) }}{{ menu_level }}{{ randomInt }}" aria-expanded="true" role="button">{{ item.title }}</a>
+        <a{{ link_attributes }} aria-controls="{{ item.title|replace({' ': ''}) }}{{ menu_level }}{{ randomInt }}" aria-expanded="true" role="button">{{ item.title }}</a>
```

Comment:
Let's use the `clean_class` filter here, the spaces are not the only characters that need to be replaced.

Please apply the change in all other instances that use `replace()`.

Also, check if it's possible to use [`clean_unique_id`](https://www.drupal.org/node/3363551) instead of the menu level and the random number to ensure a unique id.